app/src/androidTest/java/com/adammcneilly/toa/di/FakeRepositoryModule.kt

```diff
@@ -0,0 +1,39 @@
+package com.adammcneilly.toa.di
+
+import com.adammcneilly.toa.core.data.local.RoomTaskRepository
+import com.adammcneilly.toa.core.di.RepositoryModule
+import com.adammcneilly.toa.login.domain.repository.DataStoreTokenRepository
+import com.adammcneilly.toa.login.domain.repository.DemoLoginRepository
+import com.adammcneilly.toa.login.domain.repository.LoginRepository
+import com.adammcneilly.toa.login.domain.repository.TokenRepository
+import com.adammcneilly.toa.task.api.TaskRepository
+import dagger.Binds
+import dagger.Module
+import dagger.hilt.components.SingletonComponent
+import dagger.hilt.testing.TestInstallIn
+import javax.inject.Singleton
+
+@Module
+@TestInstallIn(
+    components = [SingletonComponent::class],
+    replaces = [RepositoryModule::class]
+)
+abstract class FakeRepositoryModule {
```

Comment:
We ended up not using this (or rather, bindTokenRepository is just binding the prod version), but keeping for education sake. 

Let's add KDocs.